### PR TITLE
feat(videos): convert the videos page into react component

### PR DESCRIPTION
**Pull request by QC-L**
I have completed the following functions:

* convert the videos page into react component
* change _data/videos.yml -> website/data/videos.yml
* add a few styles

See #1522

### CODE REVIEW

**Comment by QC-L** [comment]
https://deploy-preview-1524--babel.netlify.com/videos.html

**Comment by abiduzz420** [comment]
Nice work @QC-L 
I have few suggestions which can be taken into account relating to styling:
- [ ] In the `CategoryInfo`, there is lot of empty space. We can either reduce the empty space or increase the font sizes. I prefer the latter option (Decreasing the bottom margin)
- [ ] Make the `VideoItem` component center aligned. They don't even out well on the right.
I like the idea of `text-muted` for links. Looks sweet!

**Comment by hzoo** [comment]
Awesome, feel free to remove the corresponding old video files that I referenced in the issue

**Comment by QC-L** [comment]
@hzoo It has been deleted.

**Comment by hzoo** [comment]
Can you check https://babeljs.io/docs/community/videos/, it looks like it looks different when you resize (not center aligned on mobile) - I think that's the same point made ^

**Comment by QC-L** [comment]
@hzoo OK, I'm going to amend it

**Comment by QC-L** [comment]
@abiduzz420 OK, let me try.

**Comment by QC-L** [comment]
@hzoo Please try again.

File: website/static/css/video.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
+.babel-video {
+  border: 1px solid rgba(0, 0, 0, .2);
+  border-radius: .25rem;
+  height: 100%;
+  overflow: hidden;
+}
+
+.page-header {
```
Comment (hzoo) [inline website/static/css/video.css]: One issue I'm seeing is that you have styles that could be re-used across the other pages like in https://github.com/babel/website/pull/1530 - I would think video.css would only apply to that page (or that docusarus isolates each css but that's not the case)

Comment (QC-L) [reply website/static/css/video.css]: I initially felt that the style would be adjusted, so the style was not added to babel.css. I'm going to amend it.

**Pull request merged**